index.js

```diff
@@ -53,35 +52,35 @@ app.use(express.json());
 // IDEAS METHODS
 
 app.get('/ideas', (req, res) => {
-    let sortedIdeasList;
-
+    let sortedIdeas;
+    const ideas = getIdeas();
     if (req.query.sortBy === 'all') {
```

Comment:
What does it mean to sort by `all`?

Reply:
It means that it will show all the ideas. So I renamed it to allIdeas.

Reply:
The `sortBy` parameter will _always_ return all ideas - only sorted differently depending on the value - that's how I understand it. If that's the case, then `allIdeas` still doesn't communicate the sort order that you're asking for

Reply:
`req.query.sortBy === 'ideas'`?

Reply:
![Screenshot_2](https://github.com/VladimirKarin/Fund-Raising-Project-Server-Side/assets/105012251/b455b685-2a67-450e-9c30-dff006e0a8e4)
Yes, that's what I've ment

Reply:
It would read as: `sort ideas by ideas` - what does it mean to sort ideas by ideas?

Reply:
Perhaps the word sortBy is unnecessary?
I don't know If I'm wright, but I've tried another way. ) I suspect, that it's not the right one, but still, it works! )) 
![Screenshot_2](https://github.com/VladimirKarin/Fund-Raising-Project-Server-Side/assets/105012251/5c49146f-f7c3-4db2-aca7-5e59a42f837d)

Reply:
The route name `/ideas` communicates: "get ideas".

The parameter name `sortBy` communicates: "get ideas sorted by".

And now you need to pass a value: "sort ideas by <what>" - the `what` part is what's missing.

`get ideas sorted by total donation sum` - makes sense, but `get ideas sorted by ideas` - makes no sense.

What's the intention of `req.query.sortBy === 'ideas'` - what ideas do you expect to get back?

Reply:
I understand that, but I can't think of anything right now to be honest. 
Line 57, 58 should return a list of all the ideas that there is. ALL ideas. perhaps I don't need theparameter sortBy, but which do I need then?

Reply:
That's exactly right: if you need to get all ideas, then why do you need to pass `sortBy` parameters at all?

Reply:
So the way I've did last time is wrong?  There is no parameters.

Reply:
You're checking that there're no parameters at all:
```js
    if (Object.keys(req.query).length === 0) {
        sortedIdeas = getIdeasUtil();
    }
```

So what it will return if I make a query like this?
```
GET /ideas?hello=world
```

Reply:
How about htis ? 
If no parametrs, GET method returns all ideas.
![Screenshot_3](https://github.com/VladimirKarin/Fund-Raising-Project-Server-Side/assets/105012251/eb611e8c-c662-4a17-bcf0-8c77eb80e21c)

Reply:
Much better, well done.